docs/lib/project-setup/fragments/android/project-setup/51_provisionBackend.md

```diff
@@ -0,0 +1 @@
+<!-- Intentionally empty file -->
```

Comment:
?

Reply:
In the iOS version, we manually add both amplifyconfiguration.json and awsconfiguration.json to the project -- in android we are using amplify tools to achieve this, so i just left the file empty for now -- because i wasn't sure if we were using amplify tools or not for android.

Reply:
Not going to update.